x/cronos/keeper/precompiles/relayer.go

```diff
@@ -91,6 +93,10 @@ func init() {
 			relayerGasRequiredByMethod[methodID] = 61781
 		case Timeout:
 			relayerGasRequiredByMethod[methodID] = 104283
+		case RegisterPayee:
+			relayerGasRequiredByMethod[methodID] = 39000
```

Comment:
https://github.com/crypto-org-chain/cronos/issues/1666

Reply:
I can try approach to the cost of cosmos tx, but these would also be static number right?

Reply:
maybe no better way, it's already in a `init` function.